components/SubmissionForm.vue

```diff
@@ -18,23 +19,27 @@
     <span
       class="mb-2 text-neutral-600 text-sm font-normal font-['Noto Sans JP']"
     >{{ $t('submitPage.healthcareProfessionalName')}}</span>
+    <p v-show="!isValidLastName || !isValidFirstName" class="text-currentColor text-xs font-['Noto Sans JP']">{{ $t('submitPage.lastNameValidation')}}</p>
```

Comment:
nit: you can give these an id or class to name these so the next reader would know these are error messages